deployment/helm/human-connection/Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "0.1.0"
+appVersion: "0.2.2"
```

```suggestion
appVersion: "0.3.0"
```
Now. We should keep that in sync somehow. Maybe with `yarn run release`?

---

I could try that out... actually, we had some complaints that the version number is not correct in production ... I think I didn't run it properly